plugins/woocommerce-admin/client/products/fields/attributes/attributes.tsx

```diff
@@ -28,9 +29,24 @@ export const Attributes: React.FC< AttributesProps > = ( {
 
 	return (
 		<AttributeControl
-			attributeType="regular"
 			value={ attributes }
 			onChange={ handleChange }
+			onModalClose={ ( attribute ) => {
+				if ( attribute ) {
+					return;
+				}
+				recordEvent( 'product_add_options_modal_cancel_button_click' );
```

Comment:
This Tracks event name should be `product_add_attributes_modal_cancel_button_click`.

Also, this event should only be recorded when the modal is actually canceled. It currently gets logged every time the modal is closed, either by "Cancel" or "Add"